lib/gssapi/krb5/acquire_cred.c

```diff
@@ -71,6 +71,33 @@ __gsskrb5_cred_store_find(OM_uint32 *minor_status,
     return GSS_S_COMPLETE;
```

Comment:
is `__gsskrb5_cred_store_find()` now not used and can be deleted?

Reply:
It's used in `lib/gssapi/krb5/store_cred.c`, and only there.  I should move it there and make it a static, or maybe just copy `store_get()` there and use that instead.